scripts/quests/sandoria/Spice_Gals.lua

```diff
@@ -37,9 +37,7 @@ quest.sections =
             onEventFinish =
             {
                 [724] = function(player, csid, option, npc)
-                    if option == 1 then
-                        quest:begin(player)
-                    end
+                    quest:begin(player)
```

Comment:
Does this quest have a dialogue window? Like yes or no to accepting the quest. 

If we don't check for an option you would still accept the quest even if you said no.

Reply:
No, not that it seems.